applications/coreutils/mkdir.cpp

```diff
@@ -23,6 +25,12 @@ static CommandLine cmdline = CMDLINE(
     "Create the DIRECTORY(ies), if they do not already exist.",
     "Options can be combined.");
 
+void print_path(const char *path)
+{
+    if (verbose)
+        printf("%s\n", path);
```

Comment:
You should add a little message like in the original command like : 
`mkdir: created directory foo`